src/start/input_cp2k_motion.F

```diff
@@ -862,6 +866,68 @@ RECURSIVE SUBROUTINE create_geoopt_section(section, location, label, description
       CALL section_add_keyword(section, keyword)
       CALL keyword_release(keyword)
 
+      ! collects keywords related to Space Group Symmetry
+      CALL keyword_create( &
+         keyword, __LOCATION__, name="KEEP_SPACE_GROUP", &
+         description="Keep the space group symmetry operation during optimization.", &
+         usage="KEEP_SPACE_GROUP yes", &
+         default_l_val=.FALSE., lone_keyword_l_val=.TRUE., repeats=.FALSE.)
+      CALL section_add_keyword(section, keyword)
+      CALL keyword_release(keyword)
+
+      ! collects keywords related to precision for finding the space group
+      CALL keyword_create( &
+         keyword, __LOCATION__, name="EPS_GEO", &
+         description="Accuracy required for symmetry detection", &
+         usage="EPS_GEO {REAL}", &
+         default_r_val=1.e-4_dp, repeats=.FALSE.)
+      CALL section_add_keyword(section, keyword)
+      CALL keyword_release(keyword)
+
+      ! collects keywords related to precision for finding the space group
+      CALL keyword_create( &
+         keyword, __LOCATION__, name="EPS_SPGR", &
+         description="Accuracy required for symmetry detection", &
```

Comment:
`EPS_GEO` and `EPS_SPGR` have the same descriptions. Maybe you need only one? Also these keyword names are not very self-explanatory. How about `EPS_SYMMETRY`?

Reply:
One keyword is used by SPGLIB, the other in my routines. Originally, the idea was to be able to have different thresholds to make it easier to identify the space group of a set of coordinates with low accuracy (e.g. PDB files). It also helped me pass the spglib check to be able to test my routines. I agree that, at this stage, it does not make sense anymore to have both. I will take care of it.